tools/hifiasm/test-data/hifiasm-out1-raw.gfa

```diff
@@ -1,7 +0,0 @@
-A	utg000001l	0	-	m64011_190901_095311/31787056/ccs	0	16420	id:i:3	HG:A:a
-A	utg000001l	15714	-	m64012_190920_173625/158859888/ccs	0	11221	id:i:25	HG:A:a
-A	utg000001l	22204	-	m64012_190920_173625/146736917/ccs	0	11872	id:i:70	HG:A:a
```

Comment:
Ah shit, this is not good, isn't it? Why does this disapear?

Reply:
You're right, that is bad, I'm investigating.